ui/src/components/common/Breadcrumbs/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useMemo } from "react";
+import { useLocation, Link } from "react-router-dom";
+import MUIBreadcrumbs from "@mui/material/Breadcrumbs";
+import Typography from "@mui/material/Typography";
+
+import "./style.css";
+
+export function Breadcrumbs() {
+  const location = useLocation();
+
+  const crumbs = useMemo(() => {
+    const { pathname } = location;
+    if (pathname === "/") {
+      // Namespaces (home)
+      return (
+        <Typography
+          data-testid="namespace-breadcrumb"
+          className="Breadcrumbs-typ"
+          sx={{ fontWeight: 600 }}
+        >
+          Namespaces
+        </Typography>
+      );
+    }
+    const pathParts = pathname.split("/");
+    switch (pathParts.length) {
+      case 5: // pipeline view
```

Comment:
can we test it with different paths.

Reply:
i covered different paths in different `it`'s